azure/packages/azure-local-service/package.json

```diff
@@ -51,7 +51,7 @@
 		"typescript": "~4.5.5"
 	},
 	"engines": {
-		"node": ">=16.16.0"
+		"node": ">=18.17.1"
```

Comment:
this looks like its in a published package, so this is a breaking change, and far too new of a release to require all users of it to have.

Reply:
maybe do this bump to a big less new version of 18 on the next branch.